apps/chatbots/views.py

```diff
@@ -25,7 +35,44 @@ def get_queryset(self):
         return queryset.filter(pipeline__isnull=False)
 
 
-# TODO: New chatbot to be implemented as part of #1307
-class CreateChatbot(BaseExperimentView, CreateView):
-    def create_experiment(self):
-        return None
+class BaseChatbotView(BaseExperimentView):
+    template_name = "chatbots/chatbot_form.html"
+    form_class = ChatbotForm
+    active_tab = "chatbots"
+
+    def get_success_url(self):
+        return reverse("chatbots:edit", args=[self.request.team.slug, self.object.pipeline.id])
+
+
+class CreateChatbot(BaseChatbotView, CreateExperiment, CreateView):
+    title = "Create Chatbot"
+    button_title = "Create"
+    permission_required = "experiments.add_experiment"
+
+    def get_success_url(self):
+        return reverse("chatbots:edit", args=[self.request.team.slug, self.object.pipeline.id])
+
+
+@login_and_team_required
+@permission_required("experiments.view_experiment", raise_exception=True)
+def single_chatbot_home(request, team_slug: str, experiment_id: int):
+    return base_single_experiment_view(
+        request, team_slug, experiment_id, "chatbots/single_chatbot_home.html", "chatbots"
+    )
+
+
+class EditChatbot(LoginAndTeamRequiredMixin, TemplateView, PermissionRequiredMixin):
```

Comment:
Might be good to navigate back to the single chatbot home page after editing the pipeline. Either by having a  "done" button or including breadcrumbs. I think breadcrumbs might be a good idea either way